rpc/server.py

```diff
@@ -94,9 +94,13 @@ def create_tables() -> dict:
 
 @jsonrpc.method("clear_account_and_transactions_tables")
 def clear_account_and_transactions_tables() -> dict:
-    result = clear_db_tables(["current_state", "transactions"])
-    app.logger.info(result)
-    return {"status": result}
+    msg = MessageHandler(app, socketio)
+    result = None
+    try:
```

Comment:
I would remove this try and then make that `clear_db_tables` behaves correctly, so first check if there are tables, if that is the case then clear, else do nothing.